common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3239,6 +3249,11 @@
         "message" : [
           "TRANSFORM with SERDE is only supported in hive mode."
         ]
+      },
+      "VIEW_OPERATION" : {
```

Comment:
I'm not sure we need this error class. DS v2 does not support view yet.

Reply:
I will replace `UNSUPPORTED_FEATURE.VIEW_OPERATION ` with `EXPECT_TABLE_NOT_VIEW. NO_ALTERNATIVE`